Program.cs

```diff
@@ -454,7 +454,7 @@ string Link(string uid, bool linkFromGroupedType, bool nameOnly = false, bool li
                            extMethod)
                         : false));
                 if (method == null)
-                    str.AppendLine($"* {extMethod}");
+                    str.AppendLine($"* {extMethod.replace("{", "&#123;").replace("}", "&#125;")}");
```

Comment:
Ah, the string replace method has its first letter upper case and C# is case sensitive so can you just change both of the "replace" to "Replace" so I can merge this as your PR? Thanks for the PR btw, I probably wouldn't have even known Docusaurus is introducing breaking changes otherwise.

Reply:
👍